.ebextensions/02_vpn_subnets.config

```diff
@@ -0,0 +1,144 @@
+Parameters:
+  IntegrationName:
+    Type: String
+    Description: "Name of the Integration"
+    Default: "victor-test"
+  CIDRBlock:
+    Type: String
+    Description: "CIDR block selection used to create the network components where the application is deployed: ex: 172.17.4.0/24"
+    Default: '172.17.1.0/24'
```

Comment:
how do we differentiate between test mode and live mode on the same organization and repo